cmd/transaction_submitter.go

```diff
@@ -170,8 +171,12 @@ func (c *TxSubmitterCommand) Command(submitterService TxSubmitterServiceInterfac
 			metricsServeOpts.MonitorService = &tssMonitorSvc
 
 			// Inject server dependencies
+			tssDatabaseDSN, err := router.GetDNSForTSS(globalOptions.DatabaseURL)
+			if err != nil {
+				log.Ctx(ctx).Fatalf("Error getting TSS database DSN: %v", err)
+			}
+			submitterOpts.DatabaseDSN = tssDatabaseDSN
```

Comment:
Here I change the `tss` CLI command by updating the database DNS with the TSS schema (`search_path=tss`).